randovania/games/dread/logic_database/header.json

```diff
@@ -1225,6 +1225,12 @@
                 "description": "Many floors can be clippped through with some or all of the following items: Speed Booster, Flash Shift, and Grapple Beam. See https://dreadwiki.hijumpboots.com/en/tricks/clipping",
                 "require_documentation_above": 0,
                 "extra": {}
+            },
+            "SlopeClimb": {
+                "long_name": "Climb Sloped Surfaces",
+                "description": "With precise movement, Flash Shift can be used to climb Sloped surfaces.",
```

Comment:
I don't think this description should only mention Flash Shift, as this trick would definitely include things like Artaria Water Slope Skip.

This should also mention the fact you can use free-aim to jump higher off of slopes. I'd recommend something like:
`It is possible to gain height on sloped surfaces with good movement. This may require usage of techniques/items such as free-aim, Flash Shift, or Phantom Cloak.`

Reply:
I ended up doing this, but I also added climbing the slopes in EMMI Zone Spinner and the bottom part of the Waterfall. The former was already under a trick with Spin Boost, while the former was previously trickless. Both remain trickless with Space Jump.